src/Poseidon/Package.hs

```diff
@@ -83,7 +83,7 @@ data PoseidonYamlStruct = PoseidonYamlStruct
     { _posYamlPoseidonVersion :: Version
     , _posYamlTitle           :: String
     , _posYamlDescription     :: Maybe String
-    , _posYamlContributor     :: [ContributorSpec]
+    , _posYamlContributor     :: Maybe [ContributorSpec]
```

Comment:
There is generally a redundancy in a type of the sort `Maybe [Type]`, so of a maybe of a list, since an empty list already signals a "Nothing". I think we can simplify this code. I'll take a stab at that. That there is now a redundancy can be seen in the fact that the two states of this value `Just []` and `Nothing` are semantically equivalent, but differently expressed, which is not ideal.